Files/Views/Settings.xaml.cs

```diff
@@ -43,6 +46,7 @@ protected override void OnNavigatedFrom(NavigationEventArgs eventArgs)
         {
             var currentView = SystemNavigationManager.GetForCurrentView();
             currentView.BackRequested -= OnBackRequested;
+            Window.Current.CoreWindow.Dispatcher.AcceleratorKeyActivated -= CoreDispatcher_AcceleratorKeyActivated;
```

Comment:
Can this be handled in the GoBack method?

Reply:
Better yet, can we just declare the keyboard accelerator in the xaml so we don't have to register the event from code behind?

Reply:
I tried this method first. There is a problem, especially visible on the widgets page .
The selection remains in keyboard mode and selects a block.
I tried to move the KeyboardAccelerator tag, use handle, nothing works.
Microsoft recommends this method to add a GoBack shortcut ([link](https://docs.microsoft.com/en-us/windows/uwp/design/basics/navigation-history-and-backwards-navigation#support-access-keys)).

![Settings](https://i.ibb.co/LC5jSJj/settings.png)

Reply:
The selection rectangle is the correct behavior when you navigate using the keyboard. In fact, this another reason to implement it this way.

Reply:
You are right. I do it.

Reply:
@yaichenbaum This method generates an error with GoBack. It compiles, but at runtime the enum cannot be parsed. I do not know why. However, it works for the other keys.

Reply:
@yaichenbaum Using KeyboardAccelerator and Goback key is not possible. Maybe after pr #4858. This point is pending.

Reply:
The bug seem be in the framework. Change the .net version could fix it.

Reply:
@cinqmilleans UWP doesn't have official support for .NET 5, the open PR only changes the framework for the FullTrustProcess.

Reply:
@cinqmilleans Instead of detecting the keyboard accelerator, you can use `SystemNavigationManager.GetForCurrentView().BackRequested` which also works for the back button in tablet mode.

Reply:
@yaichenbaum Thanks I will try.

Reply:
@yaichenbaum This is the event that is currently used to detect button presses. My problem is detecting GoBack key. This cannot work with KeyboardAccelerator, but works for other keys. The method with CoreDispatcher works. I still say there is a bug with VirtualKey.GoBack in uwp.

Reply:
Were you able to get `GetForCurrentView().BackRequested` to work?

Reply:
@yaichenbaum I do not understand the question. I should not detect this event but trigger it. My problem is with VirtualKey.GoBack, which causes an error that other keys do not, although VirtualKey.GoBack exists in the enum, in the documentation, and intellisense. The event BackRequested works.

Reply:
Never mind, I thought we were implementing the back button shortcut here.